src/calculateAverage.js

```diff
@@ -37,7 +37,32 @@ function applyCalculateAverage() {
    * @param {Object} options - optional
    */
   [].__proto__.calculateAverage = function(options) {
-    // write code here
+    if (options === undefined) {
+      return this.reduce((a, b) => (a + b)) / this.length;
+    }
+
+    if (options instanceof Object) {
```

Comment:
More convenient to use `typeof variable === 'object' && variable !== null`, but I think your version is ok